tests/FakeItEasy.Tests.TestHelpers/ObjectAssertionsExtensions.cs

```diff
@@ -73,6 +73,16 @@ public static void BeAFake(this ReferenceTypeAssertions<object, ObjectAssertions
             return new MyExceptionAssertions<TExpectedException>(exception);
         }
 
+        public static ExceptionAssertions<TExpectedException> WithMessageModuloLineEndings<TExpectedException>(
+            this ExceptionAssertions<TExpectedException> assertion,
+            string expectedMessage,
+            string because = null,
+            params object[] becauseArgs)
+            where TExpectedException : Exception
+        {
+            return assertion.WithMessage(expectedMessage.NormalizeLineEndings(), because, becauseArgs);

```

Comment:
This isn't perfect, because it doesn't normalize the actual message, only the expected one. Seems to work, though... As long as we use Environment.NewLine, I think it should be OK